galaxy.yml

```diff
@@ -0,0 +1,72 @@
+### REQUIRED
+
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: pulp
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: pulp_installer
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 3.3.1
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Pulp Project Developers <pulp-dev@redhat.com>
+
+
+### OPTIONAL but strongly recommended
+
+# A short summary description of the collection
+description: A collection of Ansible roles to install & configure Pulp 3 from PyPI
+
+# Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
+# accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
+license:
+- GPL-2.0-or-later
+
+# The path to the license file for the collection. This path is relative to the root of the collection. This key is
+# mutually exclusive with 'license'
+license_file: LICENSE
+
+# A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
+# requirements as 'namespace' and 'name'
+tags:
+  - pulp
+  - pulpcore
+  - content
+  - software-repositories
+  - lifecycle-management
+  - content-delivery
+
+# Collections that this collection requires to be installed for it to be usable. The key of the dict is the
+# collection label 'namespace.name'. The value is a version range
+# L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
+# range specifiers can be set and are separated by ','
+dependencies: {}
+
+# The URL of the originating SCM repository
+repository: https://github.com/pulp/pulp_installer
+
+# The URL to any online docs
+documentation: https://pulp-installer.readthedocs.io/
+
+# The URL to the homepage of the collection/project
+homepage: https://pulpproject.org/
+
+# The URL to the collection issue tracker
+issues: https://pulp.plan.io/projects/pulp/issues?query_id=156
+
+build_ignore:
```

Comment:
This feature is only supported when running ansible-galaxy collection build with Ansible 2.10 or newer.
https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/developing_collections.rst#ignoring-files-and-folders